src/cmd/cmd.cpp

```diff
@@ -479,9 +412,12 @@ int main(int argc, char **argv)
             std::getline(std::cin, s);
             ctx.user = QString::fromStdString(s);
         }
+#if 0
```

Comment:
Hmm, why not remove this altogether?

Reply:
wait, thats not meant to be still around 🙈

Reply:
Did you look on the single commits?
I don't think that was in the final revision, I had to temporarily disable that code before I finished the refactoring in the last commit.

Reply:
I only looked at the separate commits, but apparently missed the final removal.